base/gmp.jl

```diff
@@ -518,6 +534,12 @@ iszero(x::BigInt) = x == Clong(0)
 <(i::Integer, x::BigInt) = cmp(x,i) > 0
 <(x::BigInt, f::CdoubleMax) = isnan(f) ? false : cmp(x,f) < 0
 <(f::CdoubleMax, x::BigInt) = isnan(f) ? false : cmp(x,f) > 0
+isneg(x::BigInt) = x.size < 0
+ispos(x::BigInt) = x.size > 0
```

Comment:
No not necessary at all, only I don't like this kind of lowlevel considerations when writing an algorithm; also I thought it could be used elsewhere, e.g. in "base/printf.jl".